milli/src/search/facet/filter.rs

```diff
@@ -342,12 +340,12 @@ impl<'a> Filter<'a> {
         match &self.condition {
             FilterCondition::Condition { fid, op } => {
                 let filterable_fields = index.filterable_fields(rtxn)?;
-                if filterable_fields.contains(&fid.to_lowercase()) {
+                if filterable_fields.contains(fid.value()) {
                     let field_ids_map = index.fields_ids_map(rtxn)?;
-                    if let Some(fid) = field_ids_map.id(&fid) {
+                    if let Some(fid) = field_ids_map.id(fid.value()) {
                         Self::evaluate_operator(rtxn, index, numbers_db, strings_db, fid, &op)
                     } else {
-                        return Err(fid.as_external_error(FilterError::InternalError))?;
+                        return Ok(RoaringBitmap::new());
```

Comment:
Do you think this is absolutely valid, though? That when a field can't be found in the `FieldsIdsMap` but has effectively been declared in the `filterableAttributes` it means that no document has ever been declared with this field and therefore we can return an empty set?

Reply:
I think it's valid yes